package.json

```diff
@@ -1,17 +1,44 @@
 {
-  "name": "",
+  "name": "yumnoms-reservations",
   "version": "",
   "description": "",
-  "author": "",
+  "author": "Laurence Nguyen",
   "license": "",
   "engines": {
     "node": ">=6.13.0"
   },
   "scripts": {
+    "build": "webpack -d --watch ",
+    "start": "nodemon server/server.js",
+    "test": "jest",
+    "lint": "eslint '**/*.{js,jsx}'"
```

Comment:
Out of curiosity, what is the benefit of having a lint script? My linter is working real-time; I don't have a script to start it.

Reply:
@jeckacodes i just use it as another view into any errors that may exist in my code. Instead of viewing each file in my editor to spot errors, I can have all errors pointed out to me in one place in my terminal.